src/client/shader.cpp

```diff
@@ -675,11 +593,11 @@ ShaderInfo generate_shader(const std::string &name, u8 material_type, u8 drawtyp
 			attribute mediump vec4 inVertexTangent;
 			attribute mediump vec4 inVertexBinormal;
 			)";
-		pixel_header = R"(
+		fragment_header = R"(
 			precision mediump float;
 			)";
 	} else {
-		shaders_header = R"(
+		shaders_header << R"(
```

Comment:
Let's fix this whole mess about the matrices here.
We're passing mWorld, but mWorldView, and mWorldViewProj are handled through the old hard coded values.
It's not hard to pass the right matrices: world, view, proj. See the part in #10686